dist/index.js

```diff
@@ -7134,6 +7134,8 @@ const runTestsUsingCommandLine = async () => {
   cmd.push('run')
   if (headless) {
     cmd.push('--headless')
+  } else if (!headless) {
+    cmd.push('--headed')
```

Comment:
This change makes it backwards compatible, but v8+ users cannot benefit from the `headless` by default mode.